lib/Models/ImageryLayerCatalogItem.js

```diff
@@ -586,84 +588,67 @@ ImageryLayerCatalogItem.enableLayer = function(catalogItem, imageryProvider, opa
             }
 
             if (tileProviderError.timesRetried === 0) {
+                // There was an intervening success, so restart our count of the tile failures.
                 tileFailures = 0;
+                tileRetriesByMap = {};
```

Comment:
retriesPerTile would be a better name.

Reply:
Also, is it definitely correct to reset it whenever there is any "intervening success"? If tile X has failed 3 times, then some other random tile succeeds, will this (incorrectly) get reset?

Reply:
It's definitely not _correct_, but the effect should be that TerriaJS is more forgiving of errors, which is closer to a :feature: than a :bug:.